guides/common/modules/proc_salt-salt-variables.adoc

```diff
@@ -0,0 +1,7 @@
+[id="salt_guide_salt_variables_{context}"]
+= Salt Variables
+
+Similar to Puppet smart class parameters, you can configure Salt Variables within {Project}.
```

Comment:
```suggestion
You can configure Salt Variables within {Project}.
```
I think it's best not to create the illusion of dependency on Puppet knowledge

Reply:
Applied.